lib/zold/commands/remote.rb

```diff
@@ -234,6 +240,9 @@ def update(opts, deep = true)
       end
     end
 
+    # @todo #292 Select the strongest nodes
```

Comment:
@botanicus let's throw an exception here for the user. You don't know when the puzzle will be resolved, maybe in a year. All this time the user will see an empty response from the gem. Not good, right?